Services/DataBaseService/DbModels/DataBaseContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DataBaseService.DbModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Reflection;
+
+namespace DataBaseService
+{
+    public class DataBaseContext: DbContext
+    {
+        public DbSet<DbUser> Users { get; set; }
+
+        private readonly IConfiguration configuration;
+
+        public DataBaseContext(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            var connectionStringTradingStation = configuration.GetConnectionString("TradingStationString");
```

Comment:
What if connecting would be with errors? Do we catch this somewhere?

Reply:
done